PKGBUILD

```diff
@@ -21,7 +21,7 @@ url="https://github.com/filebot/filebot"
 license=('GPL')
 install=$_pkgname.install
 depends=('java-openjfx' 'jre8-openjdk' 'fontconfig' 'chromaprint')
-makedepends=('ant' 'ivy')
+makedepends=('ant' 'ivy' 'jdk8-openjdk')
```

Comment:
Are you sure that JDK is definitely needed for successful build?

Reply:
I did not have it installed and "archlinux-java status" only showed "java-8-openjdk/jre" as an option. This lead to numerous errors during execution of the PKGBUILD because in it, you force the default java jdk to be set to "java-8-openjdk"...

I have not investigated whether it works without your hardcoded setting the default java to java-8-openjdk.

Reply:
Ok. Will not build without JDK and will not run after uninstalling JDK.

This one merged.